### PR TITLE
Production guidelines and standing orders

### PR DESCRIPTION
**Pull request by saty9**

### CODE REVIEW

**Comment by saty9** [comment]
awaiting formatting

**Comment by rphi** [comment]
Added formatting

File: StandingOrders.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+### 7.0 Procedure
+
+__7.1__ The proposers of all shows and all persons standing for election have the right to a short speech (duration to be specified by the Chair), followed by questions as directed by the Chair.
+
+### 8.0 Voting
+
+A Single Transferable Vote, or STV, shall be used in all cases where a preference vote is required. The voting and counting procedure is detailed below. For simplicity, ‘candidate’ shall refer to any person, show or proposal standing in an election. In any preference vote a decision against all candidates is valid and shall be indicated by the words ‘none’ or ‘no-one’, as appropriate. In any vote where only one candidate stands, a STV may be performed between the candidate and ‘none’ or ‘no-one’. This is equivalent to a simple YES/NO/ABSTAIN majority vote, where ABSTAIN is neither a YES, nor a NO, and will only be counted towards ensuring that quorum is met.
```
Comment (rphi) [inline StandingOrders.md]: Why is this duplicated?

Comment (saty9) [reply StandingOrders.md]: shouldn't be ill re-locate original wording

Comment (saty9) [reply StandingOrders.md]: actually what is duplicated?

Comment (rphi) [reply StandingOrders.md]: This, and the paragraph next in the document shares a lot of wording. May just be a typo in the document however.

Comment (saty9) [reply StandingOrders.md]: yeah although they are the same and this seems to be what the original says (aside from last line of 8.0.1 which is from an amendment

Comment (rphi) [reply StandingOrders.md]: Fair enough

**Pull request merged**